src/components/Summary/GenericSummary.js

```diff
@@ -42,7 +42,7 @@ const GenericSummary = ({
         <ErrorMessage key={error}>{error}</ErrorMessage>
       ))}
       <Formik
-        initialValues={{privacy: false}}
+        initialValues={{}}
```

Comment:
Formik requires initial values to be specified for all the fields that are managed by it, otherwise you get react warnings that mention either:

- `undefined` is not a valid value for a field (inputs must have a string value, typically, for checkboxes this translated to the `checked` prop)
- form field is changing from uncontrolled to controlled, which is an anti-pattern

So I think we should just include `privacy` and `truth` as values here (mapping to the appropriate form fields) with the default false, even if they are not relevant for this particular form.

In the future, I think we should leverage the react-router datarouters where we can configure a `loader` at the route level. This should then consult the API endpoint and provide that data, so the GenericSummary component can then get the configuration from `useLoaderData` hook. However, that's too involved too tackle now, so that's why I'd opt for just including them in a hardcoded way for now.